exercise3.js

```diff
@@ -0,0 +1,7 @@
+// Implement a JavaScript function to find the sum of all elements in an array recursively.
+
+
+function sumArray(arr){
+    if(arr.length === 0) return 0
+    return arr[0] + sumArray(arr.slice(1))
```

Comment:
The function name is fine here, but the argument variable name is not quite descriptive!
You could consider to use something like `numbers` with an `s` to indicate that there are more than one number, hence an array!

Also add some space before the return statement!
```suggestion
function sumArray(numbers){
    if(numbers.length === 0) return 0
    
    return numbers[0] + sumArray(numbers.slice(1))
```

But also in this case, we don't really need this line 
`if (numbers.length === 0) return 0` 
Because if we don't add it, the function would return `0` for empty arrays anyways!